tests/cluster/tests/24-links.tcl

```diff
@@ -1,11 +1,7 @@
 source "../tests/includes/init-tests.tcl"
 
-test "Create a cluster with two single-node shards" {
-    create_cluster 2 0
-}
-
-test "Cluster should start ok" {
-    assert_cluster_state ok
+test "Create a cluster with 10 two-node shards" {
+    create_cluster 10 10
```

Comment:
Two things:
1) We want to mostly be writing tests in the new framework. Those tests are located under https://github.com/redis/redis/tree/unstable/tests/unit/cluster. 
2) Do we need 10 primaries and 10 replicas to be initialized?